app/views/transferConfirmation.scala.html

```diff
@@ -0,0 +1,29 @@
+@()(implicit messages: Messages)
+@main(Messages("transferConfirmation.header")) {
+    @defining(play.core.PlayVersion.current) { version =>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
```

Comment:
Is it correct that the back link will take the user back to the Dashboard page from the Transfer Confirmation page? The mock wireframes take the user back to the Transfer summary page: https://u6vda4.axshare.com/reviewandconfirm.html

Reply:
I think yes, for now as every page directs to the dashboard page. I think we're still waiting on the decision as to where the back button should take you.

Reply:
OK.